### PR TITLE
Add config option to set limit of tcp connections

### PR DESCRIPTION
**Pull request by szarsti**
We found, that we had to extend default ranch 'max_connections' limit. I present this patch, because others may need to customise it as well. It feels like ddb.conf is good place to put those customizations.

### CODE REVIEW

File: apps/dalmatiner_db/src/dalmatiner_db_app.erl [hunk 1]
```diff
@@ -22,8 +22,11 @@ start(_StartType, _StartArgs) ->
                     _ ->
                         100
                 end,
+    MaxConnections = application:get_env(dalmatiner_db, tcp_max_connections, 1024),
```
Comment (elvisci) [inline apps/dalmatiner_db/src/dalmatiner_db_app.erl]: Line 25 is too long:     MaxConnections = application:get_env(dalmatiner_db, tcp_max_connections, 1024),.

File: schema/dalmatinerdb.schema [hunk 2]
```diff
@@ -10,6 +10,12 @@
  [{default, 100},
   {datatype, integer}]}.
 
+%% @doc Number of allowed concurrent tcp connections.
+{mapping, "tcp_max_connections", "dalmatiner_db.tcp_max_connections",
+ [{default, 1024},
+  {datatype, integer},
+  hidden]}.
```
Comment (Licenser) [inline schema/dalmatinerdb.schema]: I would go w/o the hidden here, this seems to be a nice thing to have in general, no reason to hide it from users.

**Comment by Licenser** [comment]
Sorry for the late response here, and thanks! Looks great :)

**Comment by Licenser** [comment]
Only thing can you please re-open this against the test branch?

**Comment by szarsti** [comment]
sure, my bad.

**Comment by Licenser** [comment]
merged in #25

**Pull request closed without merging**